src/deluge/modulation/params/param_manager.cpp

```diff
@@ -556,7 +556,11 @@ void ParamManagerForTimeline::tickSamples(int32_t numSamples, ModelStackWithThre
 void ParamManagerForTimeline::nudgeAutomationHorizontallyAtPos(int32_t pos, int32_t offset, int32_t lengthBeforeLoop,
                                                                Action* action,
                                                                ModelStackWithThreeMainThings* modelStack,
+                                                               bool nudgeAutomation, bool nudgeMPE,
```

Comment:
same enum thing - it's just hard to immediately see what the flags are setting when they're only true/false